ethereum/contracts/BeefyClient.sol

```diff
@@ -0,0 +1,474 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.5;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "./utils/Bits.sol";
+import "./utils/Bitfield.sol";
+import "./utils/MMRProofVerification.sol";
+import "./ScaleCodec.sol";
+import "./utils/MerkleProof.sol";
+
+/**
+ * @title A entry contract for the BEEFY light client
+ */
+contract BeefyClient is AccessControl {
```

Comment:
seems a bit of a waste to use AccessControl for just the initialization? can we just remove it and leave initialization open?

Reply:
We should still have a step in the deployment runbook though to "manually make sure that nobody slipped a transaction in between yours" before using the bridge, though this shouldn't even matter as the bridge won't work anyway if someone slipped in a malicious transaction right, so will inevitably be detected before the bridge is used?

Is the above correct/would u agree?

Reply:
For now I've downgraded the initialization to just using `Ownable` for access control.

I think one foreseeable problem with leaving it open is that auditors will quickly pick it up, and we'll have to end up locking it down anyway.

Anyway, once we introduce governance/configuration we'll have to revisit access control again.